lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -53,15 +53,18 @@
 
 
 @retry(tries=10, jitter=(1, 2))
-def _generate_vhd_path(container_client: Any, file_name_part: str = "") -> str:
-    path = PurePosixPath(
-        f"{get_date_str()}/{get_datetime_path()}_"
-        f"{DEFAULT_VHD_SUFFIX}_{file_name_part}.vhd"
+def _generate_vhd_path(
+    container_client: Any, file_name_part: str = ""
+) -> Tuple[str, str]:
+    path_prefix = (
+        f"{get_date_str()}/{get_datetime_path()}_{DEFAULT_VHD_SUFFIX}_{file_name_part}"
     )
-    blobs = container_client.list_blobs(name_starts_with=path)
+    vhd_path = PurePosixPath(f"{path_prefix}.vhd")
+    vmgs_path = PurePosixPath(f"{path_prefix}_vmgs.vhd")
```

Comment:
For vmgs file, is the vmgs an ext name or still be the vhd ext name?

Reply:
VMGS still has .vhd extension